ddsrouter_cmake/cmake/project/read_branch_commit_hash.cmake

```diff
@@ -0,0 +1,47 @@
+# Copyright 2022 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+###############################################################################
+# Commit hash Reader macro
+###############################################################################
+
+# Load commit hash
+
+# 
+# RETURN VARIABLES:
+# - MODULE_COMMIT_HASH    : commit-hash
+
+# Arguments:
+
+
+macro(read_branch_commit_hash)
+
+    execute_process(COMMAND
+        git rev-parse --abbrev-ref HEAD
```

Comment:
This retrieve `HEAD` in case there is no git branch.
And this is the case for version tags. So I say we do not use branch name.
@juanlofer-eprosima @rsanchez15 ?

Reply:
We finally decided to remove branch from info